docker-app/requirements/requirements.txt

```diff
@@ -71,7 +71,7 @@ django==4.2.17
     #   djangorestframework
     #   drf-spectacular
     #   jsonfield
-django-allauth==0.44.0
+django-allauth==65.3.1
```

Comment:
Did you update this manually, or you used `pipcompile`? I am surprised the other dependencies are not bumped.